Doc/constraints.txt

```
@@ -9,16 +9,16 @@ babel<3
 colorama<0.5
 imagesize<1.5
 Jinja2<3.2
-packaging<24
+packaging<25
 Pygments>=2.16.1,<3
```

Sphinx has 2.17 as a lower bound, so 2.16 will never be requested
```suggestion
Pygments<3
```